lambda/templates/api_update.json.tpl

```diff
@@ -31,6 +31,16 @@
         "kms:Decrypt"
       ],
       "Resource": "${kms_arn}"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateNetworkInterface",
```

Comment:
Are these changes needed because the Lambdas have just been moved into the private subnet?

Reply:
Yep. Not sure why they're EC2 permissions but you need them to add the lambda to the VPC.